### PR TITLE
Update simulated README and fix gn_build_test_example template path.

### PR DESCRIPTION
**Pull request by krypton36**
- Update simulated README on updated instructions to build
- Update simulated README on updated instruction in generating tests
- Update correct path in gn_build_test_example script

### CODE REVIEW

File: docs/guides/simulated_device_linux.md [hunk 1]
```diff
@@ -1,100 +1,76 @@
...
+-   [ZAP Installed](../code_generation.md#installing-zap-and-environment-variables)
 
-## Generating and building the default Simulated App with Script
+## Building the default Simulated App with Script
 
 In order to utilize the app against a commissioner or controller, the app will
 need to be specifically built.
 
-1. To generate the ZAP files, and build the `chip-app1` binary completing the
+1. To build the `chip-app1` binary completing the
```
Comment (bzbarsky-apple) [inline docs/guides/simulated_device_linux.md:26]:
```suggestion
1. To build the `chip-app1` binary perform the
```

**Review by bzbarsky-apple — APPROVED** [review]
@krypton36 Note the restyle failures.

File: docs/guides/simulated_device_linux.md [hunk 2]
```diff
@@ -1,100 +1,76 @@
...
+1. Follow the instruction to build chip-tool in the [chip-tool readme](../../examples/chip-tool/README.md).
 
-2. Run this command to commission.
-    ```
-    ./out/debug/standalone/chip-tool pairing code 0x654321 MT:-24J0AFN00KA0648G00
-    ```
-    or whatever is listed on the "SetupQRCode:" line in the log output.
-3. Most tests will start at this point and now an send cluster commands with
-   chip-tool as follow.
+2. Run this command to commission with whatever is listed on the "SetupQRCode:" line in the log output..
```
Comment (cliffamzn) [inline docs/guides/simulated_device_linux.md:71]: This is missing the actual command to run and also has an extra period at the end

**Review by cliffamzn — CHANGES_REQUESTED** [review]

File: docs/guides/simulated_device_linux.md (hunk 2 above)
Comment (krypton36) [reply docs/guides/simulated_device_linux.md:71]: Thank you

**Review by cliffamzn — APPROVED** [review]

**Review by yufengwangca — APPROVED** [review]

**Pull request merged**